cypress/support/e2e.ts

```diff
@@ -0,0 +1,20 @@
+// ***********************************************************
```

Comment:
and this

Reply:
when trying to delete this file we have an error in VSCode terminal: Your project does not contain a default supportFile. We expect a file matching cypress/support/e2e.{js,jsx,ts,tsx} to exist.